apps/code-examples/project.json

```diff
@@ -82,8 +82,8 @@
       "executor": "@nx/eslint:lint",
       "options": {
         "lintFilePatterns": [
-          "apps/code-examples/src/**/*.ts",
-          "apps/code-examples/src/**/*.html"
+          "{projectRoot}/src/**/*.ts",
+          "{projectRoot}/src/**/*.html"
```

Comment:
These changes were an NX schematic. Every `project.json` file with a `lint` target has this small change.